Problems/Problems/ReverseLinkedList.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+
+namespace Problems
+{
+    
+    // Definition for singly-linked list.
+
+    public class Node { 
```

Comment:
Let's have a separate class for LinkedList data structure and methods.
Node, AddNode, PrintList, ReverseList